lib/sidekiq_prometheus.rb

```diff
@@ -82,8 +82,8 @@ def client
   # Configure SidekiqPrometheus and setup for reporting
   # @example
   #   SidekiqPrometheus.configure do |config|
-  #     config.base_labels = { service: 'images_api' }
-  #     config.custom_labels = { sidekiq_job_count: { object_klass: nil } }
+  #     config.preset_labels = { service: 'images_api' }
+  #     config.custom_labels = { sidekiq_job_count: [:object_klass] } }
```

Comment:
`custom_labels` will be deprecated (because it's not really useful) so this change isn't really necessary.

Reply:
Okay, never mind. I see know that you've re-purposed custom_labels to be the array labels that will be registered for use.

Reply:
So this then should 

```
#   config.custom_labels = { [:object_klass] }
```

Reply:
and then in line 70 change the initial value to an array:

```
 self.custom_labels = []
```

Reply:
As you mentioned over the comment. We can resolve this comment right?